vocabs/oada/isoblue/index.js

```diff
@@ -0,0 +1,72 @@
+var libvocab = require('../../../lib/vocab')('oada');
+// Only importing the essentials, more complex definitions may need more imports
+// similar to the main oada index.js this is based off of
+const {register,  override, enumSchema} = libvocab;
+
+// Custom location schema
+register('lat', override('latitude', {
+    description: 'Short form of latitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+  
+register('lng', override('longitude', {
+    description: 'Short form of longitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+
+
+// Custom CAN schema
+register('is_error_frame',  {
+    description: 'Error frame flag for can frame',
+    type: Boolean,
+});
+
+register('is_extended_frame', {
+    description: 'Extended frame flag for can frame',
+    type: Boolean,
+});
+
+register('arbitration_id', {
+    description: 'CAN ID portion of extended can frame',
+    type: Number,
+});
+
+register('dlc',  {
+    description: 'Data Length Code portion of extended can frame',
+    type: Number,
+});
+
+register('payload', {
+    description: 'String representation can frame payload in hexadecimal',
+    type: String,

```

Comment:
The CAN payload is binary data by nature and many times we do not know what they mean (typically if they are vendor specific), and cannot interpret it before uploading. I do not believe OADA supports raw binary as a data type, so this is the next best thing? Still seems a bit clunky but might not have no choice

Reply:
Per in person review with Ault, encoding in Base64